api/src/pcapi/routes/serialization/collective_stock_serialize.py

```diff
@@ -131,7 +131,7 @@ def validate_booking_limit_datetime(  # pylint: disable=no-self-argument
     ) -> Optional[datetime]:
         if (
             booking_limit_datetime
-            and "beginningDatetime" in values
+            and values["beginningDatetime"] is not None
```

Comment:
Là si la clef n'existe pas ça va planter.

La même chose avec  `values.get(beginningDatetime") is not None` serait plus fiable (et reprendrait implicitement la précédente) (j'avais eu des problemes avec la gestion des values par pydantic, il y a pas toujours ce qu'on espere y trouver)

Reply:
j'ai fait le test en visiblement il met toutes les values à None (car dans mon payload je n'envoi bien que bookingLimitDatetime mais il me disait True pour "beginningDatetime" in values) mais jepeux mettre le get pour être sur

Reply:
(ça corrigera aussi les tests)